README.md

```diff
@@ -1,20 +1,20 @@
-# Metis
+# Baseline Site Selection Tool
 
-Project Metis, named after the Greek goddess of wisdom, implements simulation tools for clinical trial enrollment. Metis is being developed for the public good with the initial goal of expediting clinical validation of vaccine candidates for COVID-19, with a focus on targeted site selection to support enhanced recruitment for vaccine research. 
+The Baseline Site Selection Tool (BSST) implements simulation tools for clinical trial enrollment. BSST is being developed for the public good with the initial goal of expediting clinical validation of vaccine candidates for COVID-19, with a focus on targeted site selection to support enhanced recruitment for vaccine research. 
 
-[Metis](https://github.com/verilylifesciences/metis/blob/main/Metis%20-%20Baseline.pdf) allows the user to input model forecasts for regional disease prevalence, as well as historical disease incidence data. One potential source of US-only model forecasts is the [CDC ensemble](https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/forecasts-cases.html). A potential source of historical data is [Google Cloud’s open repository](https://github.com/GoogleCloudPlatform/covid-19-open-data).
+[BSST](https://github.com/verilylifesciences/metis/blob/main/BaselineSiteSelectionTool.pdf) allows the user to input model forecasts for regional disease prevalence, as well as historical disease incidence data. One potential source of US-only model forecasts is the [CDC ensemble](https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/forecasts-cases.html). A potential source of historical data is [Google Cloud’s open repository](https://github.com/GoogleCloudPlatform/covid-19-open-data).
 
-# How to use Metis
+# How to use the Baseline Site Selection Tool
 
-This repository includes [Jupyter notebooks](https://jupyter.org/), a Python package, and [Dockerfiles](https://www.docker.com/) for the Metis simulation tools.
+This repository includes [Jupyter notebooks](https://jupyter.org/), a Python package, and [Dockerfiles](https://www.docker.com/) for the simulation tools.
 
 By default the notebooks run with a small amount of demo data, so you can run them anywhere that you can run Jupyter notebooks.
 
 For use of the notebooks to run simulations on your own, larger scale data, you will need to run them on a machine with more CPUs and memory, such as those available via [Terra](https://app.terra.bio/#workspaces/verily-metis/Metis-toolkit-for-vaccine-trial-planning). Please see the computational requirements listed at the top of each notebook.
```

Comment:
Can we also change the workspace name? Perhaps in a later pull request?

Reply:
Yes, can do.

Reply:
Filed https://github.com/verilylifesciences/metis/issues/27

Reply:
The package name is also Metis, is that a harder change to make?

Reply:
Not at all - easy to do! We can change that name everywhere! Please feel free to make those changes to the code and notebooks if you like. I can take care of the Terra featured workspace and the email group.

Reply:
Also, we can change the name of the GitHub repo too. The redirects work fine if someone uses an old link to GitHub.